Sources/SecureXPC/Routes.swift

```diff
@@ -7,8 +7,57 @@
 
 import Foundation
 
-/// Consistent framework internal implementation of routes that can be sent over XPC (because its Codable) and used as a dictionary key (because its Hashable).
-struct XPCRoute: Codable, Hashable {
+/// The entry point for creating routes needed for client to server communication.
+///
+/// Client to server communication is performed using routes. A route is a sequence of `String`s and if applicable also the message and reply types.
+///
+/// In practice a route is similar to a function signature or a server path with type safety, although it is not precisely either. Message and reply types must be
```

Comment:
> although it is not precisely either

Reads awkwardly, as "(it is not precisely) either" rather than "it is not (precisely either)"

Reply:
Thanks for the feedback, will just remove this whole portion of the sentence